### DISCUSSION TITLE
What about making an RFC to mark typst code in other languages.

### DISCUSSION THREAD

**Discussion by quank123wip** [Myriad-Dreamin/typst.ts#607] [Ideas]
Usually we use the "\$" symbol to mark LaTeX code in markdown. I think it's a good idea to support both LaTeX and typst in one unified environment. I suggest finding another symbol or something else to mark typst code and make it as a standard. Maybe we can use `#$  $#` and `#$$ $$#`? It may not be a good idea, please let me know yours if you are interested in this topic.

**Comment by Myriad-Dreamin** [comment]
Introducing a new mark syntax is not good in general, but we can have configuration to configure them. This is already supported by mathjax, which has a configuration to configure whether to identify `\[..\]`, `\(..\)`, `$..$`, or `$$..$$` in markup content.

To support latex equation, another way is to introduce some preamble typst functions, e.g.

```typ
#import "@preview/mitex:0.2.2": *
#let tex-equation = mimath
```


Then we can render latex equation by calling these functions: `#tex-equation("\max{f(x)}")`.

**Reply by quank123wip** [reply]
Maybe sometimes one user just use one type of syntax in a document. Maybe we can just specific the type you use in a custom markdown header or something else. but still, I think it may be a good idea to let user specific their choice in paragraphs? Let me know what's your idea.

**Reply by Myriad-Dreamin** [reply]
Could you leave your email for offline discussion? I'll invite you to some QQ group and discuss it in Chinese there.
